lib/msf/core/modules/external/remote_exploit_cmd_stager.erb

```diff
@@ -0,0 +1,75 @@
+require 'msf/core/modules/external/bridge'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'        => <%= meta[:name] %>,
+      'Description' => <%= meta[:description] %>,
+      'Author'      =>
+        [
+          <%= meta[:authors] %>
+        ],
+      'License'     => MSF_LICENSE,
+      'References'  =>
+        [
+          <%= meta[:references] %>
+        ],
+      'DisclosureDate' => <%= meta[:date] %>,
+      'Privileged'     => <%= meta[:privileged] %>,
+      'Platform'       => [<%= meta[:platform] %>],
+      'Payload'        =>
+        {
+          'DisableNops' => true
+        },
+      'Targets'        =>
+        [
+          <%= meta[:targets] %>
+        ],
+      'DefaultTarget'  => 0,
+      'DefaultOptions' => { 'WfsDelay' => <%= meta[:delay] %> }
+      ))
+
+      register_options([
+        <%= meta[:options] %>
+      ], self.class)
+  end
+
+  def execute_command(cmd, opts)
+    mod = Msf::Modules::External::Bridge.open(<%= meta[:path] %>)
+    mod.run(datastore.merge(command: cmd))
+    wait_status(mod)
+    true
+  end
+
+  def exploit
+    print_status("Exploiting...")
+    execute_cmdstager({:flavor  => :wget})
+  end
+
+  def wait_status(mod)
```

Comment:
This method should be pulled out into a mixin, ERB helper, or something else where it won't be copy-pasta'd among all the templates.